Source/WebCore/rendering/mathml/RenderMathMLScripts.cpp

```diff
@@ -74,9 +74,9 @@ RenderMathMLOperator* RenderMathMLScripts::unembellishedOperator() const
 
 std::optional<RenderMathMLScripts::ReferenceChildren> RenderMathMLScripts::validateAndGetReferenceChildren() const
 {
-    // All scripted elements must have at least one child.
+    // All scripted elements must have at least one in-flow  child.
```

Comment:
there should be only one space after in-flow

Reply:
Fixed